packages/charts/examples/components/demo/advanced-usage/events/event-sampler/event-sampler.component.ts

```diff
@@ -180,6 +188,8 @@ export class EventSamplerComponent implements OnInit {
         const {grid, accessors, renderer, scales, seriesProcessor} = this.getChartAttributes(this.selectedChartType);
 
         this.chart = new Chart(grid);
+        this.chartAssist = new ChartAssist(this.chart);
```

Comment:
Not a big deal, but now that we have a chart assist, we no longer need to have `this.chart` since the chart assist keeps its own public chart reference.

```suggestion
        this.chartAssist = new ChartAssist(new Chart(grid));
```